godal.go

```diff
@@ -775,6 +775,11 @@ func (ds *Dataset) CreateMaskBand(flags int, opts ...DatasetCreateMaskOption) (B
 	return Band{majorObject{C.GDALMajorObjectH(hndl)}}, nil
 }
 
+// DriverShortName returns dataset driver short name.
+func (ds *Dataset) DriverShortName() string {
+	return C.GoString(C.GDALGetDriverShortName(C.GDALGetDatasetDriver(ds.handle())))
+}
+
```

Comment:
Please expose Dataset.Driver() and Driver.Name() + Driver.ShortName() instead of this function

Reply:
Changed